src/pages/index.js

```diff
@@ -6,61 +7,22 @@ import ModalWithForm from "../components/ModalWithForm.js";
 import Card from "../components/Card.js";
 import FormValidator from "../components/FormValidator.js";
 import "./index.css";
-
-const initialCards = [
-  {
-    name: "Yosemite Valley",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/yosemite.jpg",
-  },
-  {
-    name: "Lake Louise",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lake-louise.jpg",
-  },
-  {
-    name: "Bald Mountains",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/bald-mountains.jpg",
-  },
-  {
-    name: "Latemar",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/latemar.jpg",
-  },
-  {
-    name: "Vanoise National Park",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/vanoise.jpg",
-  },
-  {
-    name: "Lago di Braise",
-    link: "https://practicum-content.s3.us-west-1.amazonaws.com/software-engineer/around-project/lago.jpg",
-  },
-];
+import Api from "../components/Api.js";
+import { _ } from "core-js/";
 
 // Buttons and Other Dom Nodes
 
 const profileEditButton = document.querySelector("#profile-edit-button");
 const profileEditModal = document.querySelector("#profile-edit-modal");
 const addCardModal = document.querySelector("#add-card-modal");
 const addCardForm = addCardModal.querySelector("#add-card-form");
-// const profileTitle = document.querySelector(".profile__title");
-// const profileSubtitle = document.querySelector(".profile__subtitle");
-// const nameInput = profileEditModal.querySelector(".modal__input_type_title");
-// const jobInput = profileEditModal.querySelector(
-//   ".modal__input_type_description"
-// );
 const profileEditForm = profileEditModal.querySelector("#edit-profile-form");
-const cardListEl = document.querySelector(".cards__list");
+// const cardListEl = document.querySelector(".cards__list");
 const addNewCardButton = document.querySelector(".profile__add-button");
-// const cardTitleInput = addCardForm.querySelector(".modal__input_type_title");
-// const cardUrlInput = addCardForm.querySelector(".modal__input_type_url");
-// const cardImageModal = document.querySelector("#card-image-modal");
-// const cardImageModalClose = cardImageModal.querySelector(".modal__close");
-// const cardImage = cardImageModal.querySelector(".card__modal_image");
-
-const section = new Section(
-  { items: initialCards, renderer: renderCard },
-  ".cards__list"
-);
-
-section.renderItems();
+const profilePictureButton = document.querySelector(".profile__picture-button");
+const profilePictureForm = document.querySelector("#profile-picture-form");
+const section = new Section({ renderer: renderCard }, ".cards__list");
+const saveButton = document.querySelector(".modal__button");
```

Comment:
`const saveButton = document.querySelector(".modal__button");
`

It's just one random save button from the top of the page


Please, delete it